pkg/core/native/native_neo.go

```diff
@@ -199,9 +208,9 @@ func (n *NEO) Initialize(ic *interop.Context) error {
 		return errors.New("already initialized")
 	}
 
-	committee := ic.Chain.GetStandByCommittee()
-	cvs := toKeysWithVotes(committee)
-	err := n.updateCache(cvs, ic.Chain)
+	committee0 := n.standbyKeys[:n.cfg.GetCommitteeSize(0)]
```

Comment:
`(n *NEO) Initialize` is called only on natives deploy, but now we have `NativesUpdateHistory`, so I'd use here `ic.Block.Index` instead of `0`.

Although it's hardly ever possible that NEO is deployed not in the genesis.

Reply:
_Very_ theoretical, but fixed.